frontend/src/pages/Assessments/Quiz.tsx

```diff
@@ -20,7 +20,7 @@ const connector = connect(
   }),
   {
     getAssessment: getAssessmentAction.request,
-    updateAssessment: (responses: AssessmentResult[], id: string) => updateAssessment.request({ responses, id })
+    updateAssessment: (responses: AssessmentResult[], id: string, isFinished: boolean, currentIdx: number ) => updateAssessment.request({ responses, id, isFinished, currentIdx })
```

Comment:
what i would do here since it's a connector would be just updateAssessment: updateAssessment.request,
then in line 44 i would do () => updateAssessment(.. .. .. )